ghost/i18n/lib/i18n.js

```diff
@@ -1,6 +1,6 @@
 const i18next = require('i18next');
 
-const SUPPORTED_LOCALES = ['en', 'nl'];
+const SUPPORTED_LOCALES = ['en', 'nl', 'fr'];
```

Comment:
let's make it a multi-line array declaration straight away. I imagine every single translation will be modifying this one line causing lots of conflicts.

Reply:
➕ I would agree, nice idea

Reply:
+ alphabetically sorted? 😬

Reply:
yes, good idea! Updated :)